src/capture/image/image_capturer.cpp

```diff
@@ -55,14 +55,14 @@ void ImageCapturer::ConfigureGroupedMode(const std::string& prefs)
 
 	const auto formats = split_with_empties(prefs, ' ');
 	if (formats.size() == 0) {
-		LOG_WARNING("CAPTURE: 'default_image_capture_formats' not specified; "
-		            "defaulting to 'upscaled'");
+		LOG_WARNING("CAPTURE: 'default_image_capture_formats' not specified, "
+		            "using to 'upscaled'");
 		set_defaults();
 		return;
 	}
 	if (formats.size() > 3) {
 		LOG_WARNING("CAPTURE: Invalid 'default_image_capture_formats' setting: '%s'. "
-		            "Must not contain more than 3 formats; defaulting to 'upscaled'.",
+		            "Must not contain more than 3 formats, using to 'upscaled'.",
```

Comment:
using to 'upscaled' -> using 'upscaled'?

Reply:
fixed